packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.types.ts

```diff
@@ -0,0 +1,41 @@
+import { YValueHover } from '../../index';
+
+export interface PopoverComponentProps {
```

Comment:
If this popover extends Fluent v9 Popover, then consider intersecting these props with the v9 PopoverProps.  It may help the forwardRef correctly handle HTML attributes etc.

Reply:
@srmukher Could you address this comment